### PR TITLE
ROS 2 Use-case [7775]

### PR DESCRIPTION
**Pull request by EduPonz**

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/170/)

File: docs/use-cases.rst [hunk 1]
```diff
@@ -43,4 +43,155 @@ Typical Use-Cases
...
+==================
+
+Fast-RTPS is the default middleware implementation in the `Open Source Robotic Fundation (OSRF) <https://www.openrobotics.org/>`_
+`Robot Operating System ROS 2 <https://index.ros.org/doc/ros2/>`_. This tutorial is an explanation of how to take full
+advantage for Fast-RTPS wide set of capabilities in a ROS 2 project.
+
+The interface between the ROS2 stack and Fast-RTPS is provided by a ROS 2 package
+`rmw_fastrtps <https://raw.githubusercontent.com/ros2/rmw_fastrtps/>`_. This packages is available in all ROS 2
+distributions, both from binaries and from sources. Event thought Fast-RTPS is the default ROS 2 middleware, it is still
+possible to explicitly specify it in two different ways:
```
Comment (rsanchez15) [inline docs/use-cases.rst]: I would remove this sentence: "Event thought Fast-RTPS is the default ROS 2 middleware, it is still
possible to explicitly specify it in two different ways:" and move the 2 possibilities at the end.

Comment (rsanchez15) [reply docs/use-cases.rst]: See the comment below.

File: docs/use-cases.rst [hunk 2]
```diff
@@ -43,4 +43,155 @@ Typical Use-Cases
...
+       RMW_IMPLEMENTATION=rmw_fastrtps_cpp ros2 run <package> <application>
+
+``rmw_fastrtps`` actually provides not one but two different ROS 2 middleware implementations, both of them using
+Fast-RTPS as middleware layer: ``rmw_fastrtps_cpp`` and ``rmw_fastrtps_dynamic_cpp``. The main difference between the
+two is that ``rmw_fastrtps_dynamic_cpp`` uses introspection type support at run time to decide on the
+serialization/deserialization mechanism. On the other hand, ``rmw_fastrtps_cpp`` uses its own type support, which
+generates the mapping for each message type at build time.
+
+The default ROS 2 RMW implementation is ``rmw_fastrtps_cpp``. However, it is possible to select
+``rmw_fastrtps_dynamic_cpp`` using the environment variable ``RMW_IMPLEMENTATION`` as described above.
```
Comment (rsanchez15) [inline docs/use-cases.rst]: I would move them here to be everything more clear and straightforward.
Replace "above" for "below" and that would be everything.

Comment (EduPonz) [reply docs/use-cases.rst]: done in fb916ea

File: docs/use-cases.rst (hunk 1 above)
Comment (EduPonz) [reply docs/use-cases.rst]: done in fb916ea

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/174/)

**Pull request merged**